code/modules/mob/transform_procs.dm

```diff
@@ -37,7 +37,7 @@
 
 	O.rename_self("AI",1)
 
-	INVOKE_ASYNC(GLOBAL_PROC, .proc/qdel, src) // To prevent the proc from returning null.
+	INVOKE_ASYNC(GLOBAL_PROC, GLOBAL_PROC_REF(qdel), src) // To prevent the proc from returning null.
```

Comment:
Why is this not just `QDEL_IN`

Reply:
Is this a "whyyyyy?" comment, or a "might as well change it" comment?

Reply:
Bit of both, not 100% sure if `QDEL_IN` would even work

Reply:
I'l leave a todo